src/styles/_card.scss

```diff
@@ -0,0 +1,115 @@
+@import 'variables';
+
+.card {
+  width: 200px;
+  background-color: $four-color;
```

Comment:
Ensure that the variable `$four-color` is defined in the 'variables' file. If it's not defined, this will cause a compilation error.